python_modules/dagster/dagster/_core/definitions/base_asset_graph.py

```diff
@@ -305,10 +305,11 @@ def toposorted_entity_keys(self) -> Sequence[EntityKey]:
         """Return topologically sorted entity keys in graph. Keys with the same topological level are
         sorted alphabetically to provide stability.
         """
+        sort_key = lambda e: str(e)
```

Comment:
this is annoying -- needed to do this because you can't natively sort a list of `Union[AssetKey, AssetCheckKey]`.